make/tests

```diff
@@ -92,7 +92,7 @@ do
     echo 1>&2 "${POD_NAME} container failed to reach Running state"
     exit 1
   fi
-  sleep 1
+  sleep 5
```

Comment:
@mook-as It's not a random change :)

Reply:
Great! Why is it here? :D

Reply:
Because 1 second it too fast. I noticed sometimes it failed before the pod was ready.

Reply:
:+1: (I just wanted _some_ sort of reason to show that it's not random :stuck_out_tongue:)

Reply:
I didn't run that beautiful sed command in this file this time.